api/v1alpha1/stage_types.go

```diff
@@ -741,6 +741,8 @@ type VerificationInfo struct {
 	// AnalysisRun is a reference to the Argo Rollouts AnalysisRun that implements
 	// the Verification process.
 	AnalysisRun *AnalysisRunReference `json:"analysisRun,omitempty" protobuf:"bytes,3,opt,name=analysisRun"`
+	// CompleteTime is the time at which the Verification process was completed.
+	CompleteTime *metav1.Time `json:"completeTime,omitempty" protobuf:"bytes,6,opt,name=completeTime"`
```

Comment:
Wondering if this should not be:

```suggestion
	// FinishTime is the time at which the Verification process finished.
	FinishTime *metav1.Time `json:"finishTime,omitempty" protobuf:"bytes,6,opt,name=finishTime"`
```

Both because start/finish seems a better pair, and because I think "complete" would imply it ended _successfully_. While it also appears to be set in case of failures.

Reply:
Ah, reading more it appears to come from the `Completed` method on the AnalysisRun, which also has a nested `FinishedAt`. 😬

Looking back, I like `StartedAt` and `FinishedAt` more than what I introduced with `StartTime`. So I guess you can just pick your poison here...

Reply:
I prefer `_time` because `_at` is a little be ambiguous to me (in perspective of date or time), so I'll go with `finishTime`